### ISSUE TITLE
Improve clarity on effect of "Low Power Mode" on "Background App Refresh"

### ISSUE THREAD

**Issue by markusthamm** [corona-warn-app/cwa-app-ios#650]
## Current Implementation
I have installed the app on my iPhone SE today. The initial installation-readme shows a hint to the "Background App Refresh" / "Hintergrundbetrieb". To me it is not obvious that activating the "Low Power Mode" ("Einstellungen" / "Batterie" / "Stromsparmodus") disables the "Background App Refresh" for the app. In my understanding this is needed for the risk calculation. So this should be running once a day.

(Side note: I am too lazy to substitute all the remaining German fragments below. You can see from above what the English translation is)

## Suggested Enhancement
To make things simple. I would suggest that the initial documentation tells the user he should
- either disable the "Stromsparmodus" once a day
- or should run (which actions?) every day in the evening

## Expected Benefits
I see a risk, if users run the "Stromsparmodus" all day. This way the background-processing would never run. Or does this automatically run, once you charge your iPhone's battery?
Then we would not really have this issue. But in this case the documentation is still misleading.

**Comment by thomasaugsten** [comment]
My knowledge is the exposure notification background task has special priorities also in the Stromsparmodus this is guaranteed by the exposure notification entitlement.

**Comment by markusthamm** [comment]
I have the same understanding (now). But I still see a gap here how the system behaves and what the documentation (at least the one of the initial start-up screen) implies. I will have another look into the architecture documentation. I will also clone the iOS-related repo and maybe add a PR directly for the documentation there.
I will post an update here once I am done with the stuff I have just mentioned.

**Comment by mynchau** [comment]
Thanks markusthamm for your issue. We'll get it added to our FAQ to ensure clarity. 
Best regards
MC 
Corona Warn-App Open Source Team

**Comment by markusthamm** [comment]
Thanks for adding it to the FAQ-section. 

After some more thinking I came to the conclusion that I would rephrase the current phrasing inside the _privacy-policy.html_ from
"<p>Die App nutzt den Hintergrundbetrieb (also wenn Sie die App nicht gerade aktiv nutzen), um Ihr Risiko automatisch zu ermitteln und den Status eines registrierten Tests abfragen zu können. Wenn Sie den Hintergrundbetrieb im Betriebssystem Ihres Smartphones deaktivieren, müssen Sie alle Aktionen in der App selbst starten.</p>"
to
"<p>Die App nutzt den Hintergrundbetrieb (also wenn Sie die App nicht gerade aktiv nutzen), um Ihr Risiko automatisch zu ermitteln und den Status eines registrierten Tests abfragen zu können. Wenn Sie den Hintergrundbetrieb im Betriebssystem Ihres Smartphones deaktivieren **(z.B. durch Einschalten des Stromsparmodus)**, müssen Sie die **gerade genannten** Aktionen in der App selbst starten.</p>"

Since I am not familiar with the localization-mechanisms on iOS I am hesitating to open a PR myself.
So I leave it up to you to change the wording here.

Should I close this issue now or are you doing it?

**Comment by Lucurus** [comment]
Just FYI, Apple translates _Stromsparmodus_ to [Low Power Mode](https://support.apple.com/en-us/HT205234) and _Hintergrundaktualisierung_ to [Background App Refresh](https://support.apple.com/en-qa/HT202070) (If those are the two features you are referring to?).

**Comment by GPclips** [comment]
Hello community,

the issue has been addressed in the following FAQ: https://www.coronawarn.app/de/faq/#close_app

I will proceed and close the issue now.

Thanks,
LMM

Corona-Warn-App Open Source Team
